src/wp-includes/block-styles-on-demand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ *
+ * Load Block Styles on Command in Classic Themes
+ *
+ * Delaying the output of CSS until after the content is loaded, this way we know what blocks will be on the page, and only load the needed CSS.
+ *
+ * @package WordPress
+ * @subpackage Theme_Compat
+ * @since 6.9
+ */
+function always_load_block_styles_on_demand_init() {
+	if ( wp_is_block_theme() || ! function_exists( 'wp_should_output_buffer_template_for_enhancement' ) ) {
```

Comment:
No need for the function existe check in core since the function will always exist:

```suggestion
	if ( wp_is_block_theme() ) {
```